neo4j-jdbc-bolt/src/test/java/org/neo4j/TestcontainersCausalCluster.java

```diff
@@ -125,7 +125,7 @@ private static Neo4jContainer getNeo4jContainer(WaitStrategy waitForBolt,
                                                                       ClusterInstanceType instanceType,
                                                                       Map<String, Object> neo4jConfig,
                                                                       String name) {
-        Neo4jContainer container = (Neo4jContainer) new Neo4jContainer("neo4j:3.5-enterprise")
+        Neo4jContainer container = (Neo4jContainer) new Neo4jContainer("neo4j:3.5-enterprise") // TODO: parameterize this?
```

Comment:
not sure if this one should be parameterized or not

Reply:
I think that we can use the same approach that we have here:

https://github.com/neo4j-contrib/neo4j-jdbc/blob/fae9eeb895755a703d814def94c2ddd5a0870fd8/neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/BoltNeo4jAuthenticationIT.java#L49

Reply:
I just hope that the cluster configuration settings have not changed since 3.5 😨

Reply:
I mean by using the method `neo4jImageCoordinates()`

Reply:
> I just hope that the cluster configuration settings have not changed since 3.5 😨

Maybe we're lucky!

Reply:
Turns out this class is used only by a single, ignored, test class: `BoltRoutingIT`. I migrated it anyway.